addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -10,6 +10,8 @@
                 'o-temporary': attachment and attachment.isTemporary,
                 'o-viewable': attachment and attachment.isViewable,
             }" t-att-title="attachment ? attachment.displayName : undefined" t-att-data-attachment-local-id="attachment ? attachment.localId : undefined"
+                t-att-aria-label="attachment ? attachment.displayName : undefined"
+                role="group"
```

Comment:
keep single line whenever possible

also `attachment and ...` instead of `? :`

Reply:
- Joined the two extra lines. This was already a multiline tag, though.
- Changed by `attachment and attachment.displayName`